### ISSUE TITLE
Possibly add some world/spawn presets

### ISSUE THREAD

**Issue by iGalaxyYT** [Minehut/Issues#156]
### Expected behavior
When you do /reset-world, you should add an option to create new world or use a world preset. There could be a library of presets made by Minehut staff or donated by the community.

### Actual behavior
/reset-world just resets the world like normal.

### Detailed steps to reproduce this behavior
Not needed

### Server name
Not needed

### Link to server logs (/dl log)
Not needed

**Comment by iGalaxyYT** [comment]
Reopened the feature request because MatrixTunnel wanted me to :P

**Comment by tjhubz** [comment]
Yeah if they simply moved the world manager from the minehut menu to the book in your hotbar in the lobby it would be cool.
